deploy/role.yaml

```diff
@@ -11,14 +11,47 @@ rules:
   - ""
   resources:
   - pods
-  - services
-  - services/finalizers
-  - endpoints
+  verbs:
+  - get
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - create
+  - delete
+- apiGroups:
+  - ""
+  resources:
+  - secrets
```

Comment:
why operator need to get secrets?

Reply:
because ibm-block-csi-external-provisioner-role need it

Reply:
so why it should be here?
this is the operator deployment and not the csi-controller.   do i miss something?

Reply:
as mentioned below, the operator role needs all the rules of the driver, including the csi-external-provisioner

Reply:
1. Yes, I'm sure. It is easy to understand the reason: the role is to limit the permission of the operator, if I can create a new role in the operator and give more permissions, then there is a big security issue - in fact I can do anything!
4. Good question. I thought about it before. But the operator role has more permissions than the driver, if we use the operator role for the driver, the driver will have too big power.

Reply:
thanks for the answers
its clear now
Base on your detail it LGTM